src/utils/permissions-utils.js

```diff
@@ -14,15 +17,16 @@ export function canMove(entity) {
   const networkedTemplate = entity && entity.components.networked && entity.components.networked.data.template;
   const isCamera = networkedTemplate === "#interactable-camera";
   const isPen = networkedTemplate === "#interactable-pen";
-  const isEmoji = networkedTemplate === "#interactable-emoji";
+  const spawnerTemplate =
+    entity && entity.components["super-spawner"] && entity.components["super-spawner"].data.template;
+  const isEmoji = spawnerTemplate === "#interactable-emoji";
```

Comment:
Should this be `networkedTemplate === "#interactable-emoji" ||  spawnerTemplate === "#interactable-emoji"`? Because I think this change implies that you can't move other peoples' emoji

Reply:
That was indeed the case, though I just saw this comment and just pushed an update to address that :)